airbyte-integrations/bases/connector-acceptance-test/acceptance-test-docker.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env sh
+
+set -e
+
+die () {
+  echo "$1" 1>&2
+  exit 1
+}
+
+[ -n "$CONFIG_PATH" ] || die "Missing CONFIG_PATH"
+
+ROOT_DIR="$(git rev-parse --show-toplevel)"
+CDK_DIR="$ROOT_DIR/airbyte-cdk/python"
+CONNECTOR_TAG="$(grep connector_image $CONFIG_PATH | head -n 1 | cut -d: -f2 | sed 's/^ *//')"
+CONNECTOR_TAG_DEV="$CONNECTOR_TAG:dev"
+CONNECTOR_NAME="$(echo $CONNECTOR_TAG | cut -d / -f 2)"
+CONNECTOR_DIR="$ROOT_DIR/airbyte-integrations/connectors/$CONNECTOR_NAME"
+CONNECTOR_SUBDIR="$CONNECTOR_DIR/$(echo $CONNECTOR_NAME | sed 's/-/_/g')"
+BUILD_DIR=$(mktemp -d)
+
+if [ -n "$LOCAL_CDK" ] && [ -f "$CONNECTOR_DIR/setup.py" ]; then
+  # Copy the connector files & CDK to the build directory
+  cd "$BUILD_DIR"
+  cp $CONNECTOR_DIR/setup.py .
```

Comment:
why not copy the whole connector directory and `cd` into it instead of cherry picking files?

Reply:
There's some cruft in there like venvs etc. that, if copied into the Docker container, can cause pip to pick up a cached version of the CDK instead of installing it locally. I haven't identified exactly which file/directory/etc. was the culprit, and thought that we'd probably want to minimize the extra baggage that we copy to the container anyway, so just went with a cp of what we need. But I'm happy to keep digging and copy the whole thing if we want to mimic the local environment more closely, rather than just the CDK.